svg/iD-sprite/operations/operation-change-date-range.svg

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   version="1.1"
+   x="0"
+   y="0"
+   width="20"
+   height="20"
+   viewBox="0 0 20 20"
+   id="svg136"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:svg="http://www.w3.org/2000/svg">
+  <defs
+     id="defs140" />
+  <path
+     d="M10,1 L9,2 L9,3.062 C5.056,3.556 2,6.922 2,11 C2,15.418 5.582,19 10,19 C14.418,19 18,15.418 18,11 C18,8.533 16.881,6.341 15.125,4.875 L13.719,6.281 C15.114,7.38 16,9.086 16,11 C16,14.314 13.314,17 10,17 C6.686,17 4,14.314 4,11 C4,8.024 6.159,5.567 9,5.094 L9,6 L10,7 L13,4 L10,1 z"
+     fill="currentColor"
+     id="path132" />
+  <path
+     id="path4001"
+     style="stroke-width:0.681386"
+     d="M 8.5,9.5 V 16 h 1 l 1,-4.5 H 15 v -1 z" />
```

Comment:
This is a new icon based on operation-rotate.svg but with clock hands instead of a dot in the middle. I considered using `temaki-clock` for consistency with the Clock preset, but the clock hands seem a little off-center and the clock face is too large compared to the other validation fix icons.